docs/reference.rst

```diff
@@ -175,6 +176,10 @@ implement both :meth:`~AbstractStorage.load_session` and
    *httponly* -- cookie's http-only flag, :class:`bool` or ``None`` (the
    same as ``False``).
 
+   *encoder* -- json serializer, :class:`func`
+
+   *decoder* -- json deserializer, :class:`func`
```

Comment:
Please use `versionadded`: http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded and specify required signatures for encoder/decoder

Reply:
Would we bump the version first? I'm not entirely sure what the correct way to specify signatures in sphinx is. :func:`encoder(obj)` but that just generates `encoder(obj)()`.
I haven't really used sphinx before so I'm most certainly missing something.

Reply:
Next version will be 2.3, please use this number.

Replace :class:\`func\` with 
```
*encoder* -- session serializer.
A callable with the following signature: `def encode(param: Any) -> str: ...`.
Default is :func:`json.dumps`.

```